python/ray/tests/test_get_locations.py

```diff
@@ -126,3 +127,9 @@ def task():
     # TODO(chenshen): this is a result of converting int -1 to unsigned int;
     # should be fix by https://github.com/ray-project/ray/issues/16321
     assert location["object_size"] == 2 ** 64 - 1
+
+
+if __name__ == "__main__":
```

Comment:
Does this conflict with #25801?

Reply:
I'll update the other PR to include this.